### PR TITLE
why: add origin of the term etcd

### PR DESCRIPTION
**Pull request by fanminshi**
explain the meaning behind the term etcd.

### CODE REVIEW

**Comment by heyitsanthony** [comment]
who's asking this though?

**Comment by fanminshi** [comment]
@heyitsanthony mostly me. i have always wonder what does etcd mean, and xiang told me about that yesterday. we think it will be a good add to faq since a lot people might wonder the same thing.

**Comment by xiang90** [comment]
@fanminshi 

There is a todo for "why etcd". Probably we should start this doc now. This what does etcd stand for explanation should belong there.

**Comment by xiang90** [comment]
See https://github.com/coreos/etcd/blob/master/Documentation/docs.md#learning

**Comment by fanminshi** [comment]
@xiang90 I'll move it

**Comment by xiang90** [comment]
@fanminshi ping. can you fix this pr?

**Comment by fanminshi** [comment]
@xiang90 yeah, i'll get it finished today.

**Comment by fanminshi** [comment]
ALL fixed. PTAL

**Comment by xiang90** [comment]
lgtm. need language checking from @heyitsanthony

File: Documentation/learning/why.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Why etcd
 
+The term "etcd" originated from two ideas, unix "/etc" folder and "d"istibuted system. The "/etc" folder is a place to store configuration data for a single system whereas etcd stores configuration information for large scale distributed system. So a "d"istributed "/etc" store is "etcd", "/etc" + "d"istributed.
```
Comment (heyitsanthony) [inline Documentation/learning/why.md]: s/term/name
s/unix/the unix/
s/distributed system/distributed systems/

> So a "d"istributed "/etc" store is "etcd", "/etc" + "d"istributed.

Hence, a "d"istributed "/etc" is "etcd".

**Comment by fanminshi** [comment]
all fixed. PTAL @heyitsanthony

**Comment by heyitsanthony** [comment]
lgtm

**Pull request merged**